retrofit2-clients/src/main/java/com/palantir/remoting1/retrofit2/Retrofit2ClientBuilder.java

```diff
@@ -50,15 +52,27 @@ public Retrofit2ClientBuilder(ClientConfig config) {
     @Override
     public <T> T build(Class<T> serviceClass, String userAgent, List<String> uris) {
```

Comment:
This will change default behavior that you would expect if a service was created with just Retrofit, which kind of sucks. __On the other hand Retrofit would not accept URL and just  throw some exception, right?__ In such case it is probably not a big deal to accept more URLs.